README.md

```diff
@@ -272,6 +272,11 @@ Sign3Intelligence.getInstance(this).getIntelligence(object : IntelligenceListene
 
 
 ## Changelog
+### 3.1.1
+ - Issue resoved of dlopen failed: library "libintelligence.so" not found
+ - Fixed the 'Supported FPS ranges cannot be null' issue.
```

Comment:
Please add point :

1. Fixed crash issue in android 6 & 7 devices.
2. Sign3 APIs now require minSdk 23 to run, apps targeting below SDK 23 will have to enclose Sign3 APIs within if condition.